img/k/kickex.com.svg

```diff
@@ -0,0 +1 @@
315-4.107-4.11-8.948-7.353-14.308-9.645A45.292 45.292 0 0 0 44.957 0 44.915 44.915 0 0 0 27.45 3.546a43.803 43.803 0 0 0-14.266 9.645c-4.106 4.108-7.348 8.952-9.64 14.315A44.604 44.604 0 0 0 0 44.978c0 6.055 1.21 11.98 3.545 17.516 2.247 5.363 5.49 10.163 9.64 14.315 4.106 4.109 8.948 7.352 14.308 9.645A44.755 44.755 0 0 0 45 90c6.095 0 11.974-1.21 17.507-3.546 5.36-2.25 10.159-5.493 14.308-9.645 4.107-4.108 7.35-8.952 9.64-14.315A44.82 44.82 0 0 0 90 44.978c-.086-6.054-1.297-11.98-3.631-17.515ZM72.363 72.398c-6.57 6.574-15.087 10.51-24.25 11.201l-.087-26.468 10.159 6.487c.518.346 1.08.476 1.642.476a3.043 3.043 0 0 0 2.594-1.427c.907-1.427.475-3.373-.951-4.282l-14.914-9.514c-.043 0-.086-.043-.086-.043-.044-.044-.087-.044-.13-.087 0 0-.043 0-.043-.043-.043 0-.043-.044-.086-.044-.044 0-.087-.043-.13-.043 0 0-.044 0-.044-.043-.043 0-.043 0-.086-.043-.043 0-.087-.043-.13-.043h-.173c-.043 0-.086-.044-.13-.044h-.906c-.043 0-.13 0-.173.044h-.13c-.043 0-.086.043-.13.043-.043 0-.086.043-.129.043-.043 0-.086.043-.13.043-.043 0-.086.043-.173.043h-.043c-.043 0-.043.044-.086.044-.043.043-.087.043-.13.086 0 0-.043 0-.043.044l-15.519 8.995c-1.47.865-1.988 2.768-1.124 4.238a3.067 3.067 0 0 0 2.68 1.557c.519 0 1.081-.13 1.556-.432l10.85-6.314.087 26.727c-9.207-.692-17.767-4.627-24.337-11.244-7.392-7.266-11.455-16.997-11.455-27.377 0-6.011 1.34-11.806 3.933-17.04l22.22 12.672-10.591 5.709a3.081 3.081 0 0 0-1.254 4.195 3.132 3.132 0 0 0 2.723 1.644c.476 0 .994-.13 1.47-.39l15.519-8.39s.043 0 .043-.043c.043 0 .043-.043.086-.043.044-.044.087-.044.173-.087.044 0 .044-.043.044-.043.043-.043.086-.043.086-.087 0 0 .043-.043.086-.043l.13-.13.043-.043a.57.57 0 0 0 .13-.173l.043-.043.044-.043.043-.044.043-.043.086-.086c.044-.044.044-.087.087-.13 0 0 0-.043.043-.043v-.044l.043-.043c0-.043.044-.086.044-.086 0-.044.043-.044.043-.087 0 0 0-.043.043-.043 0-.043 0-.043.043-.087v-.043c0-.043.043-.086.043-.13v-.173c0-.043.044-.086.044-.13v-.129c0-.043 0-.13.043-.173v-.389l-.043-17.948c0-1.73-1.384-3.071-3.113-3.071s-3.112 1.384-3.069 3.114l.043 12.585L13.271 22.62c1.254-1.773 2.68-3.503 4.28-5.06 7.305-7.31 17.031-11.374 27.406-11.374s20.1 4.022 27.406 11.374c1.6 1.6 3.026 3.287 4.28 5.103-.044 0-.087.043-.13.043L54.467 35.334l.043-12.585c0-1.73-1.383-3.114-3.07-3.114-1.685 0-3.111 1.384-3.111 3.07l-.044 17.949v.389c0 .043 0 .13.044.173v.13c0 .043.043.086.043.13v.172c0 .044.043.087.043.13v.043c0 .043 0 .043.043.087 0 0 0 .043.043.043 0 .043.044.043.044.086 0 .044.043.087.043.087l.043.043v.043s0 .044.043.044c.043.043.043.086.087.13 0 .043.043.043.086.086l.044.043.043.043.043.044.043.043c.043.043.087.13.13.173l.043.043.13.13.043.043c.043.043.086.043.086.087.044 0 .044.043.087.043.043.043.087.086.173.086.043 0 .043.044.086.044 0 0 .044 0 .044.043l15.518 8.39c.476.26.951.39 1.47.39 1.08 0 2.161-.606 2.723-1.644a3.082 3.082 0 0 0-1.253-4.195l-10.591-5.709 21.873-12.542c.086-.043.13-.087.217-.13a38.776 38.776 0 0 1 3.933 16.997c0 10.466-4.064 20.197-11.369 27.506Z" fill="#0CC466"/></svg>
```

Comment:
There's a few problems with the SVG that's other of our control.

Basically, the SVG that KickEX uses is one of the worst vector graphics I've ever seen.
I don't understand how they managed to fail that bad but if you zoom in on the arrow head you can see something which looks more like the Gadsden flag's snake than the head of an arrow.

Actually... let me present to you the Gadsden flag by KickEX. A story in four parts:
First we copy the head of the arrow:
![screenshot_1643154752](https://user-images.githubusercontent.com/3535780/151081131-c1e57d40-f9db-47d1-ac88-969b378b72ff.png)

Then we paste it:
![screenshot_1643155782](https://user-images.githubusercontent.com/3535780/151081103-ecd0ba4f-f4b4-4a20-a072-9267c6e8b71f.png)

Add a mouth:
![screenshot_1643155853](https://user-images.githubusercontent.com/3535780/151081113-614dc532-d4ef-4567-8f9f-f31c83e839a2.png)

A voilà!
![](https://upload.wikimedia.org/wikipedia/commons/f/fc/No_Step_On_Snek_%28Gadsden_Flag%29.svg)

TLDR; we can't use this SVG because it would make vector graphic designers cry.

I did some googling to try and find a good PNG to replace the SVG with but couldn't find one. Maybe you have better luck? If not then I can take the JPEG they use on their twitter and crop out the other white space.

Reply:
I wonder how much I could earn if I sold the snek as an NFT :thinking: 
Maybe the KickEX staff are interested
![snek.png.mov.exe](https://user-images.githubusercontent.com/3535780/151083508-85f5268c-de0b-46a8-852d-867cfda111f9.png)

Reply:
🤣🤣🤣 👍 🧐